tests/unit/Config.js

```diff
@@ -311,4 +311,208 @@ describe('Config', () => {
         const { config } = require('../../lib/Config');
         assert.strictEqual(config.overlayVersion, 0);
     });
+
+    describe('azureArchiveLocationConstraintAssert', () => {
+        const { azureArchiveLocationConstraintAssert } = require('../../lib/Config');
+
+        it('should return undefined if azureStorageEndpoint is missing', () => {
```

Comment:
> return undefined

this function does not actually return "anything" : so we don't really care what it returns, only that is actually succeeds (as in "does not assert")

→ should just be "should succeed when azureStorageEndpoint is missing", and no reason to have any assert (just call the function)